lightningd/peer_control.c

```diff
@@ -1200,10 +1203,39 @@ static struct command_result *json_setchannelfee(struct command *cmd,
 		   NULL))
 		return command_param_failed();
 
-	// TODO: write this stuff
+	// find the channel
+	peer = peer_from_json(cmd->ld, buffer, idtok);
+	if (peer) {
+		channel = peer_active_channel(peer);
+		if (!channel)
+			return command_fail(cmd, LIGHTNINGD,
+				    "Could not find active channel of peer with that id");
+	} else {
+		struct command_result *res;
```

Comment:
Variable declarations should be at the top of the function.

Reply:
same here, in-function declaration done same file function:
```C
static struct command_result *json_close(struct command *cmd, ...)
```

as i see stuff like this, is it not my task to correct other code? if so, who does the cleanup sometime?